apps/nextjs-website/declarations.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'swagger-ui' {
+  const SwaggerUI: any;
+  export default SwaggerUI;
+}
```

Comment:
I don't get why is necessary to define a custom declaration instead of using types from npm libraries (https://www.npmjs.com/package/@types/swagger-ui), i think i am missing somethig what is the reason of this choice?

Reply:
<img width="1663" alt="image" src="https://github.com/pagopa/developer-portal/assets/16854782/85acd242-bb9a-4f0e-b014-e787fce4cd82">

This error is thrown in `apps/nextjs-website/src/helpers/swagger.ts`, when i try to use `System` prop so, as you can see in the screenshot, i have added the declaration file to fix it. 
I don't install the swagger-ui types  because it doesn't contains the prop that i need.